framework/src/mfem/bcs/MFEMVectorFunctorDirichletBCBase.C

```diff
@@ -0,0 +1,26 @@
+#ifdef MFEM_ENABLED
+
+#include "MFEMVectorFunctorDirichletBCBase.h"
+
+InputParameters
+MFEMVectorFunctorDirichletBCBase::validParams()
+{
+  InputParameters params = MFEMEssentialBC::validParams();
+  params.addRequiredParam<MFEMVectorCoefficientName>(
+      "vector_coefficient",
+      "Vector coefficient specifying the values variable takes on the boundar. A coefficient can "
+      "be any of the "
+      "following: a variable, an MFEM material property, a function, or a post-processor.");
+  return params;
+}
+
+// TODO: Currently assumes the vector function coefficient is 3D
```

Comment:
I should have deleted this in a previous PR. Tidying up now that I'm thinking of it.
```suggestion
```